dask/cli.py

```diff
@@ -0,0 +1,68 @@
+from dask import __version__
+
+try:
+    import click
+except ImportError as e:
+    msg = (
+        "The Dask CLI requires click to be installed.\n\n"
+        "Install with conda or pip:\n\n"
+        " conda install click\n"
+        " pip install click\n"
+    )
+    raise ImportError(msg) from e
+
+
+@click.group
+@click.version_option(__version__)
+def cli():
+    """Dask command line interface."""
+    pass
+
+
+@cli.command
+def docs():
+    """Open Dask documentation in a web browser."""
+    import webbrowser
+
+    webbrowser.open("https://docs.dask.org")
+
+
+@cli.group("info")
+def info():
+    """Information about your dask installation."""
+    pass
+
+
+@info.command()
+def versions():
+    """Print versions of Dask related projects."""
+    from dask.utils import show_versions
+
+    show_versions()
+
+
+def register_third_party(cli):
```

Comment:
My guess is we don't want users interacting with this method. If that's the case, let's prepend a leading underscore to signal the method is private

Reply:
ah yes, fixed